src/components/Header.jsx

```diff
@@ -1,11 +1,15 @@
 import React from "react";
 import { NavLink } from "react-router-dom";
-import { Menu, Segment } from "semantic-ui-react";
+import { Menu, Segment, Flag, Icon } from "semantic-ui-react";
 import { useDispatch } from "react-redux";
 import { getArticles } from "../modules/getArticles";
+import i18n from "../i18n";
```

Comment:
I would import this module in `index.js` that is the highest level. Please try that and let me know if you are okay

Reply:
we tried but then our tests failed. it said i18n is not defined in the header component

Reply:
I think that is fixable with 
```
const { t, I18n } = useTranslation();
```

Reply:
or perhaps: 
```
const { t, i18n } = useTranslation();
```

Note the lowercased `i` 

Or a combination thereof...

Reply:
The second one worked! Thanks